core/src/stored/append.cc

```diff
@@ -342,18 +520,19 @@ bool DoAppendData(JobControlRecord* jcr, BareosSocket* bs, const char* what)
     // Restore the original data pointer.
     jcr->sd_impl->dcr->rec->data = rec_data;
 
-    if (bs->IsError()) {
+    if (auto* error = handler.error()) {
```

Comment:
as `error` is passed to `Jmsg2()` which isn't type-checked at all, I would feel safer with a `const char*` instead of an `auto*`

Reply:
Makes sense!